alacritty/src/config/window.rs

```diff
@@ -126,15 +128,15 @@ impl Default for Decorations {
 
 /// Window Dimensions
 ///
-/// Newtype to avoid passing values incorrectly
+/// Newtype to avoid passing values incorrectly.
 #[serde(default)]
 #[derive(Default, Debug, Copy, Clone, Deserialize, PartialEq, Eq)]
 pub struct Dimensions {
-    /// Window width in character columns
+    /// Window width in character column.
     #[serde(deserialize_with = "failure_default")]
     columns: Column,
 
-    /// Window Height in character lines
+    /// Window Height in character line.
```

Comment:
Pretty sure these two (`column` and `line`) should still be plural. Also maybe you could lowercase `Height` to bring it in line with the above comment?
```suggestion
    /// Window height in character lines.
```

Reply:
My vim macro failed here.